sci-libs/silo/files/silo-4.10.2-mpiposix.patch

```diff
@@ -1,53 +0,0 @@
---- src/hdf5_drv/silo_hdf5.c.orig	2017-07-28 08:52:09.026638768 +0200
```

Comment:
Dropping these patches should have been done in the second commit since we should have a working package at every commit, this way if you checkout right after the first commit you're left with version 4.10.2 in the tree but no patches it requires :(

Reply:
I thought you merge all commits in 1 time

Reply:
Well yes, but the idea is we should preserve a working and functional tree at every commit. Suppose if you a new package version, but it doesn't compile, then you fix it up in another commit, the first commit isn't really complete since the package is broken, right? ;)

Reply:
Right, I will take more consideration about that in future

Reply:
...and it could also cause inconvenience if we for any reason need to revert a commit. Anyway this is just to keep in mind as a good practice when working with git, I think I'm going to merge this as is today